ms-campus-kid-subjects/DataBase/Career.cs

```diff
@@ -11,11 +11,13 @@ namespace DataBase
     public class Career
     {
         [Key]
-        public string id_career { get; set; }
-        public string career_name { get; set; }
-        public string id_faculty { get; set; }
-        [ForeignKey("id_faculty")]
-        public virtual Faculty faculty { get; set; }
+        [DatabaseGenerated(DatabaseGeneratedOption.Identity)]
+        public int Id_Career { get; set; }
```

Comment:
Could you use `camelCase` for attribute naming?
This is a standard that we are using in the project.